OMCompiler/SimulationRuntime/c/simulation/solver/initialization/initialization.c

```diff
@@ -322,9 +322,21 @@ static int symbolic_initialization(DATA *data, threadData_t *threadData)
       }
 
       if(0 == step)
-        data->callback->functionInitialEquations_lambda0(data, threadData);
+      {
+        if(data->callback->functionInitialEquations_lambda0 != NULL)
+        {
+          data->callback->functionInitialEquations_lambda0(data, threadData);
+        }
+        else
+        {
+          warningStreamPrint(LOG_INIT_HOMOTOPY, 0, "No initialEquation_lambda0 was generated. Using normal initla equation system with lambda=0 instead.");
```

Comment:
```suggestion
          warningStreamPrint(LOG_INIT_HOMOTOPY, 0, "No initialEquation_lambda0 was generated. Using normal initial equation system with lambda=0 instead.");
```

Reply:
Ups. I was to fast with merging it. I'll fix it in one of the next commits.
Or forget about it and this will be in omc forever 😆